frontend/packages/kubevirt-plugin/src/statuses/vm/vm.ts

```diff
@@ -262,6 +262,41 @@ export const getSimpleVMStatus = (
     : VM_SIMPLE_STATUS_OTHER;
 };
 
+export const getVMStatusString = ({ vm, vmi, pods, migrations }) => {
+  switch (getVMStatus({ vm, vmi, pods, migrations }).status) {
+    case VM_STATUS_V2V_CONVERSION_PENDING:
+      return 'Pending (Import VMware)';
+    case VM_STATUS_IMPORT_PENDING:
+      return 'Pending (Import)';
+    case VM_STATUS_V2V_CONVERSION_IN_PROGRESS:
+      return 'Importing (VMware)';
+    case VM_STATUS_V2V_CONVERSION_ERROR:
+      return 'Error (Import VMware)';
+    case VM_STATUS_POD_ERROR:
+      return 'Error (Pod)';
```

Comment:
Isn't it going to be confusing that the sort behaves differently than the visual perspective?

I get, that the categories might be useful. Anyway, we should rather name it `getVMStatusSortString`.

side note: we could have the categories present in `ValueEnum`

Reply:
renamed and moved to constants